src/reporting/interfaces/mappingInterfaces/CalculatedProperties.ts

```diff
@@ -0,0 +1,159 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Move all the files from `mappingInterfaces` folder up to the `interfaces` folder. I don't think we need the subfolder.

Reply:
removed mappingInterfaces folder